scripts/deploy_arm32.sh

```
@@ -0,0 +1,531 @@
+}
+
+# Check ARM32 architecture
+check_arm32_architecture() {
+    print_step "Checking ARM32 architecture"
+    
+    local arch_cmd="uname -m"
+    if [[ -n "$REMOTE_HOST" ]]; then
+        arch_cmd="ssh $REMOTE_HOST 'uname -m'"
+    fi
+    
+    local arch=$(eval "$arch_cmd" 2>/dev/null || echo "unknown")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)